tests/unit/test_scenario.py

```diff
@@ -0,0 +1,34 @@
+from unittest.mock import ANY
+
+import ops
+import pytest
+from scenario import Context, Container, Relation, State
+
+from charm import HexanatorCharm
+
+META = {
+    "name": "hexanator",
+    "requires": {
+        "ingress": {
+            "interface": "ingress"
+        }
+    },
+    "containers": {
+        "gubernator": {}
+    },
+
+}
+
+
+def test_startup():
+    ctx = Context(HexanatorCharm, meta=META)
+    container = Container(name="gubernator", can_connect=True)
+    relation=Relation(endpoint="ingress", interface="ingress", remote_app_name="ingress")
```

Comment:
The change to require some args be keyword is very new, but we think the pattern will be that the obvious ones are still positional, so we'd encourage:

```suggestion
    container = Container("gubernator", can_connect=True)
    relation=Relation("ingress", interface="ingress", remote_app_name="ingress")
```

Obviously, either will work and if you prefer being explicit I don't see anything wrong with that.

Reply:
Done